client/hotrod-client/src/main/java/org/infinispan/client/hotrod/Reproducer.java

```diff
@@ -0,0 +1,73 @@
+package org.infinispan.client.hotrod;
+
+import org.infinispan.client.hotrod.configuration.ConfigurationBuilder;
+
+import java.util.Random;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+
+public class Reproducer {
```

Comment:
Looks like an improper test; not located under src/test/ and name is not like *Test.java.